manager/manager.hpp

```diff
@@ -79,7 +79,7 @@ class Manager
      *  @param [in] bus       - sdbusplus handler
      *  @param [in] LedLayout - LEDs group layout
      */
-    Manager(sdbusplus::bus_t& bus, const GroupMap& ledLayout) :
+    Manager(sdbusplus::bus_t& bus, const LedLayout& ledLayout) :
```

Comment:
Looks like you have changed LedLayout to GroupMap in this commit "sdbusplus: use shorter type aliases
" 
and you reverted it back to LedLayout in this commit "Use the lower camel case of the C++ standard".

since GroupMap is not declared anywhere in this PR, when you compile the commits individually, you will hit compile time errors. Please avoid such changes.

Reply:
I compiled complete changes (after pulling all the commits) then after I pushed.
As You can see latest File has LedLayout and not GroupMap, I don't get your comment.